code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -314,6 +314,11 @@ func (fsh *StorageHandler) DownloadFile(
 				"error parsing the auth ticket for download: %v", err)
 		}
 
+		// we check if the file is available for download or not
+		if authToken.Available > common.Now() {
```

Comment:
@Rizary You should not save the available date in authticket. Instead it should be saved in Database